include/z64animation.h

```diff
@@ -241,21 +241,23 @@ typedef s32 (*AnimUpdateFunc)();
 
 typedef struct SkelAnime {
     /* 0x00 */ u8 limbCount;      // Number of limbs in the skeleton
-    /* 0x01 */ u8 mode;           // 0: loop, 2: play once, 4: partial loop. +1 to interpolate between frames.
+    /* 0x01 */ u8 mode;           // See `AnimationMode`
     /* 0x02 */ u8 dListCount;     // Number of display lists in a flexible skeleton
     /* 0x03 */ s8 taper;          // Tapering to use when morphing between animations. Only used by Door_Warp1.
     /* 0x04 */ void** skeleton;   // An array of pointers to limbs. Can be StandardLimb, LodLimb, or SkinLimb.
     /* 0x08 */ void* animation;   // Can be an AnimationHeader or LinkAnimationHeader.
-    /* 0x0C */ f32 startFrame;    // In mode 4, start of partial loop.
-    /* 0x10 */ f32 endFrame;      // In mode 2, Update returns true when curFrame is equal to this. In mode 4, end of partial loop.
-    /* 0x14 */ f32 animLength;    // Total number of frames in the current animation's file.
+    /* 0x0C */ f32 startFrame;    // In mode ANIMMODE_LOOP_PARTIAL*, start of partial loop.
+    /* 0x10 */ f32 endFrame;      // In mode ANIMMODE_ONCE*, Update returns true when curFrame is equal to this.
+                                  // In mode ANIMMODE_LOOP_PARTIAL*, end of partial loop.
+    /* 0x14 */ f32 animLength;    // Total number of frames in the current animation.
     /* 0x18 */ f32 curFrame;      // Current frame in the animation
     /* 0x1C */ f32 playSpeed;     // Multiplied by R_UPDATE_RATE / 3 to get the animation's frame rate.
     /* 0x20 */ Vec3s* jointTable; // Current translation of model and rotations of all limbs
     /* 0x24 */ Vec3s* morphTable; // Table of values used to morph between animations
     /* 0x28 */ f32 morphWeight;   // Weight of the current animation morph as a fraction in [0,1]
     /* 0x2C */ f32 morphRate;     // Reciprocal of the number of frames in the morph
-    /* 0x30 */ s32 (*update)();   // Can be Loop, Partial loop, Play once, Morph, or Tapered morph. Link only has Loop, Play once, and Morph
+    /* 0x30 */ s32 (*update)();   // Can be Loop, Partial loop, Play once, Morph, or Tapered morph.
+                                  // Link only has Loop, Play once, and Morph
```

Comment:
i would personally have the comment line be longer than to have a line break in the struct. we dont run the formatter o headers so the line limit isnt a concern
just a personal opinion though, open to discussion from others

Reply:
we need some agreed-on way to have longer comments on struct members

the issue with this style here is only the first line seems to appear in documentation when the mouse hovers on the struct member

is it really fine to have a very very long line? like 200 characters may be "ok", but 300 probably not

some options:

```c
    /* 0x30 */ s32 (*update)(); // Can be Loop, Partial loop, Play once, Morph, or Tapered morph. Link only has Loop, Play once, and Morph

    /* Can be Loop, Partial loop, Play once, Morph, or Tapered morph.
     * Link only has Loop, Play once, and Morph */
    /* 0x30 */ s32 (*update)();

    /*
     * Can be Loop, Partial loop, Play once, Morph, or Tapered morph.
     * Link only has Loop, Play once, and Morph
     */
    /* 0x30 */ s32 (*update)();
```